src/autogluon/bench/eval/scripts/run_generate_clean_openml.py

```diff
@@ -17,7 +17,7 @@
 @app.command()
 def clean_amlb_results(
     benchmark_name: str = typer.Argument(
-        help="Benchmark name populated by benchmark run, in format <benchmark_name>_<timestamp>"
+        None, help="Benchmark name populated by benchmark run, in format <benchmark_name>_<timestamp>"
```

Comment:
Is `benchmark_name` not required?

Reply:
I got an error if this isn't present on import I believe.

Reply:
Let me know if I did something incorrectly here

Reply:
Do you have the exact comment you ran? I believe this command in README still works
```
agbench clean-amlb-results {benchmark_name} --results-dir-input s3://{METRICS_BUCKET}/aggregated/{module}/{benchmark_name}/ --benchmark-name-in-input-path --constraints constratint_1 --constraints constratint_2 --results-dir-output {results_dir_output} 
--out-path-prefix {out_path_prefix} --out-path-suffix {out_path_suffix}
```

Reply:
Just tested on master and `agbench clean-amlb-results` should work out of the box. Let me know if you can reproduce the error.